tests/chart_tests/test_kibana.py

```diff
@@ -55,6 +55,9 @@ def test_kibana_index_defaults(self, kube_version):
         )
         common_kibana_cronjob_test(docs)
         doc = docs[0]
+        assert doc["spec"]["template"]["spec"]["nodeSelector"] == {}
```

Comment:
We need a test case here to verify that custom nodeSelector, affinity, and tolerations work as expected.